### ISSUE TITLE
Implement Testimonial Cards Section

### ISSUE THREAD

**Issue by riyaadazad** [mtspcampllc/mtspcampllc.github.io#8]
![IMG_2255](https://user-images.githubusercontent.com/34426689/73007151-fec8ee00-3dd9-11ea-8edb-b99b143e075d.jpeg)

**Comment by riyaadazad** [comment]
The following is an excerpt from [mtspcamp.com](https://www.mtspcamp.com/kids-own-words): 

Kids in their own words
   
Jamoy Pitter
"I have a lot of fun I have a lot fun and games. Skipping is so fun"
​
Delano Witter
"I have a lot of fun I play cricket and football. I learn a lot of things"
​
Rashai
"I think this is a very good fun and interactive camp it gives the kids and adults a chance to learn and have fun at the same time"
​
Marlon
"This summer has made a great impact on the students and also the adults. It shows the adult how children thinks when they are having fun. It also helps other children to interact with the adult and peers"
​
Andre
"This summer camp is a great experience for both adults and kids. It helps the kids to work together and acknowledge one another. The adults get the experience of there life working with the kids, for example adults get chances to help kids learn more activities and get more educated in the sports they do. Overall the experience is spectacular"
​
Aaron Sailsman
"The camp this year is a great experience to me. It is fun and enjoyable. I am looking forward to it next year"
​
Zodane Witter
"I love this summer camp every year. I glad to play cricket at this summer camp"
​
Tyson Stewart
"I love this summer camp every year I love to play cricket here"
​
Delan Williamson
"I have a lot of fun at this summer camp. The games that we play is fun and the equipment that we use is very interesting. I saw things that I never see before and I love to play the sporting exercise"
​
Tomoiya Christian
"It is a good thing to have this summer camp. Last year was great! Lots of children come out with great poses. I played volleyball, cricket. And basketball. I have a great time also the lunch was great and the staff member. It was awesome"
​
Tyrese Witter
"The champ this year and last year was good. This year has more thing. Than last year but we not going to has any party this year"
​
Shaquille Simpson
"Last year, I experienced the summer sports camp. I played a lot of games and had a lot of fun. Some games I played were football and volleyball. I had a lot of fun"
​
Reneil Witter
"Last year summer champ It was a very good summer champ it is so good and it was funny and fine it have a lot of fun it have football and good things it was football, basket, netball. We have picture book of all the children it to it and we have things, cricket volleyball"
​
Prom Mickela
"I love the camp, and I like the people. They help us play games. I like the things that they carry for us to play with, and I love the skipping game. They give away prizes which was exciting"
​
Deshanay Rodney
"It was ok but last year better and more fun"
​
Jadon Johnson
"The camp was a great experience. There was a lot of activities for us which we enjoyed very much. I hope the camp will continue for many more years"
​
Loshaun Witter
"The experience of the camp was great because we had lots of fun sport activities to do like basketball, and lots more"
​
Akeem Clarke
"I am so excited about the summer camp. I have so much fun. I play football, cricket and other fun games. We win prizes and candies"
​
Neko Neil
"I love this summer camp this summer camp is fun. I like this summer camp this summer camp are fun"
​
Shanghi Walters
"Last year camp was better than this year camp because last year camp had football"
​
Brenae Wright
"I love this camp very much. It teaches you a lot of different stuff like you must love your self first. There are so much games to play. When I come hear in the morning there are so much new games to play every morning"
